### PR TITLE
Allow the validate button to accept cubic

### PR DESCRIPTION
**Pull request by jake-kirkland-teton**
Closes is-cura-ui #369

### CODE REVIEW

**Comment by jake-kirkland-teton** [comment]
Let me adjust the unit tests real fast.

File: pywim/am.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ class InfillType(enum.Enum):
     triangle = 2
     triangles = 2
     cubic = 3
+    tri_hexagon = 4
```
Comment (bradymadams) [inline pywim/am.py:13]: Good idea adding a separate unsupported pattern so the unit tests can remain. Let's rename to `trihexagon` to match [Cura](https://github.com/Ultimaker/Cura/blob/master/resources/definitions/fdmprinter.def.json#L1730), which will make it a little easier to reference between Smart Slice and Cura if we ever need to.

**Review by bradymadams — CHANGES_REQUESTED** [review]
Just need to rename the trihexagon type, and then also need to increment the version (in `pywim/_version.py`)

File: pywim/am.py (hunk 1 above)
Comment (jake-kirkland-teton) [reply pywim/am.py:13]: Ah gotcha. I should have checked that. I have it changed.

**Review by bradymadams — APPROVED** [review]

**Pull request merged**